portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -52,23 +58,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String comment = request.getParameter("comment");
+        String comment = request.getParameter(Constants.COMMENT_TEXT_ID);
         long timestamp = System.currentTimeMillis();
         
         if(!comment.equals("")){
-            Entity commentEntity = new Entity("Comment");
-            commentEntity.setProperty("comment", comment);
-            commentEntity.setProperty("timestamp", timestamp);
+            Entity commentEntity = new Entity(Constants.COMMENT_ENTITY_ID);
+            commentEntity.setProperty(Constants.COMMENT_TEXT_ID, comment);
+            commentEntity.setProperty(Constants.COMMENT_TIMESTAMP_ID, timestamp);
             
-            DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
             datastore.put(commentEntity);
         }
-        response.sendRedirect("/index.html");
+        response.sendRedirect(Constants.INDEX_URL);
     }
 
     private String convertToJsonUsingGson(ArrayList<String> messages) {
         Gson gson = new Gson();
```

Comment:
Intermediate assignment to variable is unnecessary

Reply:
Done